spyder/widgets/mixins.py

```diff
@@ -327,7 +327,7 @@ def handle_sub(matchobj):
 
         return title
 
-    def show_calltip(self, signature, parameter=None):
+    def show_calltip(self, signature, parameter=None, language="python"):

```

Comment:
You probably need to pass `language` to the call to `_format_signature` inside this method.

But @goanpeca knows more about it. @goanpeca?